backend/core/src/cron/cron.service.ts

```diff
@@ -1,10 +1,47 @@
 import { Injectable } from "@nestjs/common"
 import { Cron, CronExpression } from "@nestjs/schedule"
+import { UserListQueryParams } from "src/auth/dto/user-list-query-params"
```

Comment:
You need to use relative paths when referencing other stuff in backed/core, so this should be: `../auth/dto/user-list-query-params`